docs/quickstarts/javascript/react/update-the-application.mdx

```diff
@@ -0,0 +1,128 @@
+---
+sidebar_label: Update the application
+---
+
+# Update the application
+
+To test this endpoint we're going to have to make sure the React app actually sends the authentication token to the server and requests the protected resources. To do that, we'll have to make some changes to the App.js file in our React app.
+
+```javascript
```

Comment:
this code snippet is a bit confusing - I would say "find the following code block" (and quote the first four lines), and then say "insert this code block right after the definition of the isAuthenticated variable" (and then quote the function), and remove the ". . . "